caching-service/README.md

```diff
@@ -0,0 +1,86 @@
+online-services: Example of using JDG Caching Service on OpenShift
+==================================================================
+Author: Sebastian Łaskawiec
+Level: Intermediate
+Technologies: Infinispan, OpenShift
+Summary: The `online-services` quickstart demonstrates how to connect to a deployed JDG Online Services
+Target Product: JDG
+Product Versions: JDG 7.2
+
+What is it?
+-----------
+
+The `online-services` quickstart demonstrates how to connect to to a deployed JDG Online Services.
+
+Prerequisites
+-------------
+
+In order to run this project one needs to have an access to OpenShift installation with JDG Caching Service installed. Follow [the documentation](https://github.com/jboss-container-images/jboss-dataservices-image/tree/dataservices72-dev/caching-service) how to do it.
```

Comment:
404 on this link. We don't have a dataservices72 branch

Reply:
I'll fix this when I add auth support

Reply:
Good catch! Thanks Ryan!